docs/elements.md

```diff
@@ -1703,7 +1703,15 @@ An overlay is pre-defined as a "overlay area" with a static size. By default, el
 
 ### `pl-external-grader-variables` element
 
-Displays variables that are given to the student, or expected for the student to define in externally-graded questions. The list of variables should be stored in `data['params']` and has the following format:
+Displays variables that are given to the student, or expected for the student to define in externally-graded Python questions.
+In particular, **we advise against using this element for any non-Python externally graded question**, since the way this element
+stores parameters is specifically for use with the Python autograder.
```

Comment:
It'd be nice to provide an alternative:

```suggestion
In particular, **we advise against using this element for any non-Python externally graded question**, since the way this element stores parameters is specifically for use with the Python autograder. If you want to display a table of variables for a question using a different autograder, consider using a standard HTML `<table>` instead.
```